### PR TITLE
Tadau implementation

### PR DESCRIPTION
**Pull request by caiotomazelli**

### CODE REVIEW

File: dags/destinations/gads_customermatch.py [hunk 1]
```diff
@@ -188,41 +190,58 @@ def send_data(
         print(err_msg)
         failures.append(err_msg)
     print(
-      f"There were '{len(failures)}' user rows that couldn't be processed.")
+        f"There were '{len(failures)}' user rows that couldn't be processed.")
```
Comment (Mr-Lopes) [inline dags/destinations/gads_customermatch.py:193]: Nit.: there's some space

File: dags/destinations/gads_customermatch.py [hunk 2]
```diff
@@ -188,41 +190,58 @@ def send_data(
...
     #   detail how many rows were processed, and how many operations
     #   were created
     if dry_run:
       print("Running as a dry run, so skipping upload steps.")
       return RunResult(
-        dry_run=True,
-        successful_hits=len(user_data_operations) - len(failures),
-        failed_hits=len(failures),
-        error_messages=failures
+          dry_run=True,
```
Comment (Mr-Lopes) [inline dags/destinations/gads_customermatch.py:201]: Nit.: Extra space

File: dags/destinations/gads_customermatch.py [hunk 3]
```diff
@@ -188,41 +190,58 @@ def send_data(
...
-    failures.append(response_failures)
+      response = self._offline_user_data_job_service.add_offline_user_data_job_operations(
+          request=add_user_data_request
+      )
+     
+      response_failures = self._process_response_for_failures(response)
+      failures.append(response_failures)
+      
+      self._offline_user_data_job_service.run_offline_user_data_job(
+
```
Comment (Mr-Lopes) [inline dags/destinations/gads_customermatch.py:221]: Nit.: Extra lines

**Review by Mr-Lopes — APPROVED** [review]
This looks great. LGTM. I've added only a few nits

File: dags/destinations/gads_customermatch.py (hunk 2 above)
Comment (caiotomazelli) [reply dags/destinations/gads_customermatch.py]: Done!

File: dags/destinations/gads_customermatch.py (hunk 3 above)
Comment (caiotomazelli) [reply dags/destinations/gads_customermatch.py]: Done!

File: dags/destinations/gads_customermatch.py (hunk 1 above)
Comment (caiotomazelli) [reply dags/destinations/gads_customermatch.py]: Done!

**Review by Mr-Lopes — APPROVED** [review]
LGTM

**Pull request merged**